config/config_test.go

```diff
@@ -41,3 +32,65 @@ func TestProctorURL(t *testing.T) {
 		assert.NoError(t, err)
 	}
 }
+
+func TestProctorEmailId(t *testing.T) {
+	proctorConfigFileExistedBeforeTest, proctorConfigFilePath, existingConfigFileData, proctorConfigDir := readProctorConfigFile()
+
+	EmailId := []byte("EMAIL_ID: foobar@gmail.com")
+	err := ioutil.WriteFile(proctorConfigFilePath, EmailId, 0644)
+	assert.NoError(t, err)
+
+	config.InitConfig()
+	configuredEmailId := config.EmailId()
+
+	assert.Equal(t, "foobar@gmail.com", configuredEmailId)
+
+	if proctorConfigFileExistedBeforeTest {
+		err = ioutil.WriteFile(proctorConfigFilePath, existingConfigFileData, 0644)
+		assert.NoError(t, err)
+	} else {
+		err = os.Remove(proctorConfigFilePath)
+		assert.NoError(t, err)
+
+		err = os.Remove(proctorConfigDir)
+		assert.NoError(t, err)
+	}
+}
+
+func TestProctorAccessToken(t *testing.T) {
+	proctorConfigFileExistedBeforeTest, proctorConfigFilePath, existingConfigFileData, proctorConfigDir := readProctorConfigFile()
+
+	AccessToken := []byte("ACCESS_TOKEN: access-token")
+	err := ioutil.WriteFile(proctorConfigFilePath, AccessToken, 0644)
+	assert.NoError(t, err)
+
+	config.InitConfig()
+	configuredAccessToken := config.AccessToken()
```

Comment:
We don't find diffs for file `config/config.go` for method `AccessToken()`. Why isn't it showing up in PR review when the changes exist on branch/

Reply:
That code was committed to gojektech/proctor add-auth-headers brach directly